tests/core/test_evaluation_parameters.py

```diff
@@ -306,3 +307,46 @@ def test_evaluation_parameters_for_between_expectations_parse_correctly(
             "result": {"observed_value": 3},
         }
     )
+
+
+def test_build_evaluation_parameters():
+    evaluation_parameters = {
+        "my_min": 1,
+        "my_max": 5,
+    }
+    expectation_args = {
+        "min_value": {"$PARAMETER": "my_min", "$PARAMETER.upstream_row_count": 10},
+        "max_value": {"$PARAMETER": "my_max", "$PARAMETER.upstream_row_count": 50},
```

Comment:
What is the significance of `upstream_row_count`? My rough understanding was that you could pass in a default value for evaluation params, but I would have thought that this would be called something like `"$PARAMETER.my_min"`.